src/Http2.re

```diff
@@ -52,36 +74,27 @@ module Http2Session = {
     "on";
   [@bs.send]
   external onConnect:
-    (
-      t,
-      [@bs.as "connect"] _,
-      [@bs.uncurry] (
-        (t, Stream.subtype(Buffer.t, Buffer.t, [> Net.Socket.kind])) => unit
-      )
-    ) =>
-    t =
+    (t, [@bs.as "connect"] _, [@bs.uncurry] ((t, Net.Socket.t) => unit)) => t =
     "on";
   [@bs.send]
   external onError:
     (t, [@bs.as "error"] _, [@bs.uncurry] (Js.Exn.t => unit)) => t =
     "on";
-  /** TODO: uncurry after release of bs-platform v7.3 */
   [@bs.send]
   external onFrameError:
     (
       t,
       [@bs.as "frameError"] _,
-      (~type_: int, ~errorCode: int, ~streamId: int) => unit
+      [@bs.uncurry] ((~type_: int, ~errorCode: int, ~streamId: int) => unit)
```

Comment:
Using `@bs.uncurry` for a few functions where it wasn't supported by older BS versions.